packages/@o3r/rules-engine/src/devkit/rules-engine-devtools.service.ts

```diff
@@ -31,7 +31,7 @@ export class OtterRulesEngineDevtools {
     private readonly rulesEngineService: RulesEngineRunnerService,
     @Optional() @Inject(OTTER_RULES_ENGINE_DEVTOOLS_OPTIONS) options: RulesEngineDevtoolsServiceOptions) {
 
-    const eventsStackLimit = (options || OTTER_RULES_ENGINE_DEVTOOLS_DEFAULT_OPTIONS).rulesEngineStackLimit;
+    const eventsStackLimit = (options || OTTER_RULES_ENGINE_DEVTOOLS_DEFAULT_OPTIONS as RulesEngineDevtoolsServiceOptions).rulesEngineStackLimit;
```

Comment:
Why do you need the cast here ?

Reply:
I'd like to discuss that we the team, here the compiler knows that `rulesEngineStackLimit` is never defined in `OTTER_RULES_ENGINE_DEVTOOLS_DEFAULT_OPTIONS`
But we want to make sure it's taken into account if we add it one day for some reason
That might be one reason why we wouldn't want more accurate typings